packages/drivers/routerlicious-driver/src/errorUtils.ts

```diff
@@ -12,8 +12,16 @@ import {
 } from "@fluidframework/driver-utils";
 import { pkgVersion as driverVersion } from "./packageVersion";
 
-export enum R11sErrorType {
+/**
+ * Routerlicious Error types
+ * Different error types that may be thrown by the routerlicious driver
+ */
+export enum RouterliciousErrorType {
+	/**
+	 * File not found, or file deleted during session
+	 */
 	fileNotFoundOrAccessDeniedError = "fileNotFoundOrAccessDeniedError",
+
 	sslCertError = "sslCertError",
```

Comment:
Can we document this one too?

Reply:
I'd like to document this and the above `fetchTokenError` but I didn't feel familiar enough to write a description. I'd be happy to add them in if we can find the right person to describe them 🙂